doc/dev/kdb-mount.md

```diff
@@ -0,0 +1,204 @@
+For this mode invoke the command without at most one argument:
+
+```
+kdb mount [MOUNTPOINT]
+```
+
+Here `[MOUNTPOINT]` is an optional argument.
+If it is present information about the backend mounted at `[MOUNTPOINT]` will be printed.
+Without the any arguments `kdb mount` prints basic information about all existing backends.
+
+## Import Mode
+
+Some backend plugins may require complex configuration.
+It is not always (easily) possible to express the full scope of such configurations in a series of command-line arguments.
+To solve this problem, `kdb mount` also has an import mode:
+
+```
+kdb mount -I <IMPORT_PLUGIN>:<IMPORT_FILE> [IMPORT_OPTIONS...] <MOUNTPOINT> <BACKEND_PLUGIN>
+```
+
+or
+
+```
+kdb mount --import=<IMPORT_PLUGIN>:<IMPORT_FILE> [IMPORT_OPTIONS...] <MOUNTPOINT> <BACKEND_PLUGIN>
+```
+
+> **Note:** Any arguments after `<BACKEND_PLUGIN>` will be ignored in this mode.
+> Additionally, not all options for `kdb mount` may be compatible with import mode.
+
+In this mode `kdb mount` will use the (storage) plugin `<IMPORT_PLUGIN>` to load the file `<IMPORT_FILE>`.
+The resulting `KeySet` will be passed to `<BACKEND_PLUGIN>` for validation and then used directly as the configuration for the new backend mounted at `<MOUNTPOINT>`.
+Essentially, this is like a `kdb import` directly into `system:/elektra/mountpoints` with some additional validation.
+The corresponding `kdb import` command would be (`<EM>` is the escaped form of `<MOUNTPOINT>`):
+
+```
+cat <IMPORT_FILE> | kdb import system:/elektra/mountpoints/<EM> <IMPORT_PLUGIN>`
+```
+
+> **Note:** The command above is only theoretical, since `kdb import` doesn't allow writing to keys below `system:/elektra/mountpoints` to avoid creating invalid backend configurations.
+
+## Example Usage: `backend`
+
+To create a standard file-based backend the default backend plugin `backend` can be used.
+
+As a reminder, the plugin `backend` will receive all the remaining arguments `[...]` from the command:
+
+```
+kdb mount [OPTIONS...] <MOUNTPOINT> <BACKEND_PLUGIN> [...]
+```
+
+These arguments are interpreted one by one as follows:
+
+- The first argument will be used as the relative path of the backing configuration file.
+- If an argument starts with a `/`, it is a config key for the preceding plugin.
+- Otherwise, it specifies a plugin.
+
+Arguments `<plugin>` that specify a plugin result in a new section `plugins/<plugin>` with at least the key `plugins/<plugin>/name = <plugin>` in the backend configuration.
+
+The arguments that start with a `/` must be of the form `<key>=<value>`, where `<key>` must be a cascading key name (`<value>` can be empty).
+The config key `<key>` with the value `<value>` will be assigned to the preceding plugin.
+In other words, this results in the key `plugins/<plugin>/config/<key> = <value>` being added to the backend configuration, where `<plugin>` is the last `<plugin>` that was specified.
+
+An example `kdb mount` invocation could look like this:
```

Comment:
Can you give a more simple example first? And you first need to introduce the file-based backend so that we know what we talking about.